.env.example

```diff
@@ -51,6 +52,31 @@ BUNDLE_TRACKER_ADDRESS=0xF96cd74e7EEcb93a773105269b7ef5187db30aef
 PROPOSER_ADDRESS=0xYourPublicKeyHere
 PROPOSER_KEY=0xYourPrivateKeyHere
 
+# ─────────────────────────────────────────────────────────────────────────────
+# WEBHOOK CONFIGURATION
+# ─────────────────────────────────────────────────────────────────────────────
+#
+# Used when the node (bundle proposer) sends notifications to the web app
+# whenever a bundle is proposed or finalized.
+#
+
+# Webhook endpoint URL on the web app server
```

Comment:
This webhook could be adapted to broadcast to Oya API providers and other Oya nodes in addition to OyaChat.